feconf.py

```diff
@@ -83,6 +83,10 @@
     'TextInput': {
```

Comment:
For now, we can drop this and then we can add it later when TextClassifier prediction service is completed.